features/step_definitions/chef-databag.rb

```diff
@@ -5,5 +5,5 @@
 end
 
 Given(/^I create an empty data bag '(.+)'$/) do |databag|
-  run_command_and_stop "knife data bag create #{databag} -z -c config.rb", false
+  run_command_and_stop "knife data bag create #{databag} -z -c config.rb", :fail_on_error => true
```

Comment:
would be good to have similar structure as we have in other files `{ fail_on_error: true }`

Reply:
Done